pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -137,91 +137,71 @@ popular among Racketeers as well.
+The recommend plugin detects the @tt{filetype} option based on the @(hash-lang)
 line. For example:@itemlist[
-    @item{A file starting with @code{#lang racket} or @code{#lang racket/base}
-    has @tt{filetype} equal to @tt{racket}.}
-    @item{A file starting with @code{#lang scribble/base} or @code{#lang scribble/manual}
-    has @tt{filetype} equal to @tt{scribble}.}
+    @item{A file starting with @code{#lang racket} or @code{#lang racket/base} has @tt{filetype} equal to @tt{racket}.}
+    @item{A file starting with @code{#lang scribble/base} or @code{#lang scribble/manual} has @tt{filetype} equal to @tt{scribble}.}
 ]
-Depending on which plugin you have, modifiers like @code{at-exp} may also be
-ignored, so that @code{#lang at-exp racket} is still a @tt{filetype} of
-@tt{racket}.
-
-This approach is more flexible but may lead to more work. Since each
-@(hash-lang) has its own @tt{filetype}, options, syntax highlighting, and other
-features need to be configured for each filetype. This can be done via the
-standard @tt{ftplugin} mechanism. See for example @tt{:help ftplugin-overrule}
-and @tt{:help ftplugin}: place your options for @tt{<lang>} in
-@filepath{~/.vim/after/ftplugin/<lang>.vim}
-(@filepath{$HOME/vimfiles/after/ftplugin/<lang>.vim} on MS-Windows). Similarly,
-syntax files follow the standard mechanism documented in @tt{:help syntax}.
-
-Both plugins come with configuration for Racket
-(and possibly other @(hash-lang)s) as @tt{ftplugin}s. To enable them, use the
-@tt{:filetype} command as documented in @tt{:help :filetype}. You likely want to
-turn on filetype plugins (@tt{:help :filetype-plugin-on}) and filetype indent
-plugins (@tt{:help :filetype-indent-on}).
+
+The recommended plugin comes with configuration for Racket and some other
+standard Racket languages.
+
+Many Racket languages still need syntax and indent support. If you create Vim
+support for other Racket languages, please consider contributing them to
+@hyperlink["https://github.com/benknoble/vim-racket"]{benknoble/vim-racket} so
+other Vim users will benefit.
 
 @subsection{Indentation}
 
-You can enable indentation for Racket by setting both the @tt{lisp} and
+If you use @hyperlink["https://github.com/benknoble/vim-racket"]{benknoble/vim-racket}
+and Vim version 9 or greater, improved indentation for the @tt{racket} filetype
+is configured out of the box.
+
+You can manually enable indentation for Racket by setting both the @tt{lisp} and
 @tt{autoindent} options in Vim. You will want to customize the buffer-local
 @tt{lispwords} option to control how special forms are indented. See @tt{:help
-'lispwords'}. Both plugins mentioned in @secref{vim-plugins} set this option for
-you.
-
-However, the indentation can be limited and may not be as complete as what you
-can get in Emacs. You can also use Dorai Sitaram's
+'lispwords'}. However, this indentation can be limited and may not be as complete
+as what you can get in Emacs. You can also use Dorai Sitaram's
```

Comment:
It's not clear to me whether this is an alternative to the indentation in `vim-racket`, or if this is something that could be used in addition to `vim-racket` for even better indentation support. If it is the former, one option could be for this paragraph to start: "Otherwise, you can manually ..."

Reply:
Right, "Otherwise" is clearer. I've also clarified that `scmindent` is in contrast to `lispwords`, not `vim-racket`.